fem/pgridfunc.cpp

```diff
@@ -566,7 +566,13 @@ void ParGridFunction::ProjectCoefficient(Coefficient &coeff)
 
    if (delta_c == NULL)
    {
+      (*this) = std::numeric_limits<real_t>::min();
```

Comment:
Should this initialization be done in `GridFunction::ProjectCoefficient` instead, so that the non-parallel case is also initialized? I suppose it doesn't matter, if there are undefined DOFs only in parallel, and that must be true if we do not initialize in the base class `GridFunction`.

Reply:
Line 569 forms a tandem with line 574. Therefor I like to fact that they are in close proximity.
This is a potential point of confusion and fragility.  I will document this better here and in the accompanying gridfunction memberfunction.